### PR TITLE
[v2.5.8] Release system-charts

### PR DESCRIPTION
**Pull request by pennyscissors**

### CODE REVIEW

File: charts/rancher-eks-operator/1.0.800/questions.yaml
Comment (cbron) [inline charts/rancher-eks-operator/1.0.800/questions.yaml:1]: @PennyScissors should the former chart be maxing version to be compatible with your new images.txt system ?

File: charts/rancher-gke-operator/1.0.100/values.yaml
Comment (cbron) [inline charts/rancher-gke-operator/1.0.100/values.yaml]: This will need to be `v1.0.1` before we merge

File: charts/rancher-eks-operator/1.0.800/questions.yaml
Comment (pennyscissors) [reply charts/rancher-eks-operator/1.0.800/questions.yaml:1]: Yes, if we don't add a max the old version, all the images in that chart will be added to the image list in future releases once my changes are in.

File: charts/rancher-gke-operator/1.0.100/questions.yaml
Comment (PennyScissors) [inline charts/rancher-gke-operator/1.0.100/questions.yaml:1]: Same for this chart, the previous version needs a max.

File: charts/rancher-gke-operator/1.0.100/values.yaml
Comment (pennyscissors) [reply charts/rancher-gke-operator/1.0.100/values.yaml]: cc @cmurphy

Comment (cbron) [reply charts/rancher-gke-operator/1.0.100/values.yaml]: To be clear its correct for now. GKE still has open issues. We just need to remind ourselves to cut one later.

Comment (superseb) [reply charts/rancher-gke-operator/1.0.100/values.yaml]: Is this good with `v1.0.1-rc13` or are changing it still?

File: charts/rancher-gke-operator/1.0.100/Chart.yaml
Comment (cbron) [inline charts/rancher-gke-operator/1.0.100/Chart.yaml]: I can't comment on the old thread, but we can't merge this until we release GKE image. GKE is not finished yet.

**Review by cbron — CHANGES_REQUESTED** [review]

File: charts/rancher-gke-operator/1.0.100/questions.yaml
Comment (cbron) [inline charts/rancher-gke-operator/1.0.100/questions.yaml]: @cmurphy Shouldn't this be: 
```suggestion
rancher_min_version: 2.5.8-alpha1
```

Comment (cmurphy) [reply charts/rancher-gke-operator/1.0.100/questions.yaml]: @cbron yes it should

Comment (cmurphy) [reply charts/rancher-gke-operator/1.0.100/questions.yaml]: https://github.com/rancher/system-charts/pull/455

**Review by SheilaghM — APPROVED** [review]

**Review by paynejacob — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**
